### PR TITLE
refactor: add enable status in entity vars

### PR DESCRIPTION
**Pull request by SameerPr**
## Description of the change

Add enable status as must_have on those vars which are required to create a SQL model.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by sprksh — APPROVED** [review]
LGTM, take a review from 
@yajuaryarudderstack 
 also.. he will be able to point out if anything is missing. Also, assuming that you have tested it with nil remapping scenarios

**Review by tipped73 — APPROVED** [review]
LGTM

**Review by sprksh — APPROVED** [review]

File: pb_project.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Project name
 name: shopify_features
 # Project's yaml schema version
-schema_version: 49
+schema_version: 51
```
Comment (yajuaryarudderstack) [inline pb_project.yaml:4]: As this is version 51, should we merge it after new release is out?
